Sources/Classes/BiometryService.swift

```diff
@@ -7,18 +7,59 @@
 
 import LocalAuthentication
 
+/// The protocol used to define the specifications necessary for the SDK.
 public protocol BiometryService: AnyObject {
 
+  /// A boolean value stating whether the biometric feature is enabled.
   var isEnabled: Bool { get set }
+
+  /// A boolean value stating whether the biometric feature is available.
   var isAvailable: Bool { get }
+
+  /// Indicates the type of the biometry supported by the device.
   var supportedType: BiometryType { get }
+
+  /// Fallback button title.
   var localizedFallbackTitle: String? { get set }
+
+  /// Cancel button title.
   var localizedCancelTitle: String? { get set }
+
+  /// Allows setting the default localized authentication reason on context.
```

Comment:
I think we should use `in context` instead of `on` 🙏
```suggestion
  /// Allows setting the default localized authentication reason in context.
```

Reply:
hmm, I'm following on the Apple document :D
<img width="736" alt="Screen Shot 2021-02-04 at 17 17 31" src="https://user-images.githubusercontent.com/19943832/106878744-e25dd200-670c-11eb-855b-cefb3d7a6c07.png">

Reply:
so weird ^^! Looking at the description I don't really get what does `on context` mean 😅

Reply:
lets me update a little bit about the description :D